### PR TITLE
Tokenization 1.2

### PR DESCRIPTION
**Pull request by lepidopterane-atsmith**
Updated IOexception error message in Chat, Panel does not house Chat
but actually does its job instead.

### CODE REVIEW

**Comment by aslo** [comment]
Thanks for addressing my feedback! Next time though, it would be better to make these changes in the original pull request before merging it (in this case, if you pushed this commit before merging https://github.com/vgomezASC/CodeU-Summer-2017/pull/5, you wouldn't need to open this new PR.)

**Comment by lepidopterane-atsmith** [comment]
I tried to push this to #5 , but for some reason it wouldn't let me push another commit to that end. I'll figure that out for next time, though.

File: src/codeu/chat/client/commandline/Chat.java [hunk 1]
```diff
@@ -58,7 +58,8 @@ public boolean handleCommand(String line) {
         args.add(token);
       }
     } catch (IOException e){ 
-      System.out.println("ERROR! IOException caught in Chat.handleCommand().");
```
Comment (aslo) [inline src/codeu/chat/client/commandline/Chat.java:61]: Why did you make this change here?

File: src/codeu/chat/client/commandline/Panel.java [hunk 2]
```diff
@@ -17,394 +17,44 @@
...
 import java.util.List;
-import java.io.IOException;
 
-import codeu.chat.client.core.Context;
-import codeu.chat.client.core.ConversationContext;
-import codeu.chat.client.core.MessageContext;
-import codeu.chat.client.core.UserContext;
-import codeu.chat.util.Tokenizer;
-
-public final class Chat {
```
Comment (aslo) [inline src/codeu/chat/client/commandline/Panel.java:31]: Nice job deleting this!

**Review by aslo — CHANGES_REQUESTED** [review]
Nice job overall. Just a few comments.

**Review by aslo — APPROVED** [review]

**Pull request merged**